source/agora/test/Base.d

```diff
@@ -538,8 +538,8 @@ public class TestAPIManager
         /// Private functions used for `shutdown`
         static void shutdownWithLogs (TestAPI node)
         {
-            node.printLog();
             (cast(FullNode)node).shutdown();
+            node.printLog();
```

Comment:
Weeeeeeell on the other hand we might not be able to print logs after `shutdown` because, you know, finalization.

Reply:
Finalization of what?

Reply:
The logs are static.

Reply:
Nvm, it's Monday.